docs/workflows/react-native-specific-workflow-steps/react-native-unit-test.md

```diff
@@ -26,6 +24,17 @@ The workflow steps that need to be executed before running the **React Native Un
 
 <Screenshot url='https://cdn.appcircle.io/docs/assets/BE4443-rnUnitFlowOrder.png'/>
 
+:::danger Step Rule
+
+There's one important setting that you should change. If any workflow steps fail, Appcircle automatically skips other steps and jumps directly to the Export Build Artifacts step. However, it is possible that some of your tests may fail. **If Test Report Component doesn't run, reports will not be generated.** You should turn on the following toggles so that Test Report Component always runs whether your tests fail or pass.
```

Comment:
I have two feedback for this warn:

Can you applied the my suggestion for this warn on Test Reports component document?
IMO this warn must be for React Native UI Test component, not for Test Reports
Already Test Reports has own warn message.

Reply:
Same rule available for unit test, I update the sentence and the screenshot.